gdx/src/com/badlogic/gdx/math/MathUtils.java

```diff
@@ -182,6 +182,24 @@ static public int randomSign () {
 		return 1 | (random.nextInt() >> 31);
 	}
 
+	/** Returns a random point inside the unit circle */
+	static public Vector2 randomUnitCirclePoint (Vector2 vector) {
+		float theta = random(0f, PI2);
+		float sqrt = (float) Math.sqrt(random());
+		return vector.set(sqrt * cos(theta), sqrt * sin(theta));
+	}
+
+	/** Returns a random point on the unit sphere's surface */
+	static public Vector3 randomUnitSpherePoint (Vector3 vector) {

```

Comment:
I'm not sure if I understand what you mean. The method should output uniformly distributed points on a unit sphere surface. All the points have spherical radius 1, so how they could be inside the sphere (ball), and not on the surface?